cypress/support/drawing.js

```diff
@@ -58,7 +59,8 @@ Cypress.Commands.add('goToDrawing', (menuIsClosed = false) => {
     addDefaultIconsFixtureAndIntercept()
     addSecondIconsFixtureAndIntercept()
     cy.goToMapView()
-    if (menuIsClosed) {
+    const viewportWidth = Cypress.config('viewportWidth')
+    if (viewportWidth && viewportWidth < BREAKPOINT_PHONE_WIDTH) {
```

Comment:
Do we still need the `menuIsClosed` parameter (in the future)?

Reply:
no I don't think so, I can clean that up